src/main/webapp/view.html

```diff
@@ -2,9 +2,9 @@
 <meta charset="utf-8" />
```

Comment:
why did you rename it to view.html? lecture-view.html seemed like a more descriptive name.

Reply:
File name present in URL--we want to keep them short